scripts/quests/sandoria/The_Rumor.lua

```diff
@@ -69,9 +70,10 @@ quest.sections =
             },
         },
     },
+
     {
-        check = function(player, status)
-            return status == xi.quest.status.COMPLETED
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE
```

Comment:
Dont you mean `QUEST_COMPLETED` ? The issue would be with quest.status, I would think. And we already have a `QUEST_AVAILABLE` section